src/Raven.Client/Documents/EntityToBlittable.cs

```diff
@@ -142,8 +135,8 @@ private void SetClrType(Type entityType, BlittableJsonReaderObject json)
 
             BlittableJsonReaderObject metadata;
             json.TryGet(Constants.Metadata.Key, out metadata);
```

Comment:
Or if there _are_ modifications there, or if the value is already set and we don't need to change it.